app/models/cohort.rb

```diff
@@ -15,6 +15,9 @@ class Cohort < ApplicationRecord
               message: "Choose true or false for funding cap",
             }
 
+  # TODO: remove "allow_nil" and add default value "gen_random_uuid()" and constraints into the DB after separation
```

Comment:
same comment as above, we left those intentionally not auto generated in the db: the ones that aren't surfaced in the API